.gitignore

```diff
@@ -22,3 +22,5 @@ _testmain.go
 *.exe
 *.test
 *.prof
+
+.idea
```

Comment:
This is responsability of each developer remove in his own gitignore globally, for a library this is not part of his responsabillity

Reply:
ok, I will delete this line